src/server/polaris-server.ts

```diff
@@ -61,7 +63,34 @@ export class PolarisServer {
                     config.connectionManager as PolarisConnectionManager,
                 ).getRepository(SnapshotPage);
                 const result = await snapshotRepository.findOne({} as any, id);
-                res.send(result?.getData());
+                if (!result) {
+                    res.send({});
+                } else {
+                    result.setLastAccessedTime(new Date());
```

Comment:
you don't need this, you have annotation on this

Reply:
Decorator*